ppr-ui/src/views/userAccess/QsInformation.vue

```diff
@@ -1,18 +1,93 @@
 <template>
-  <span>Qualified Supplier Information Placeholder</span>
+  <div id="qs-information">
+    <section class="qs-information-intro mt-10">
+      <p>
+        To request <strong>{{ getMhrSubProduct }}</strong> access to the Manufactured Home Registry, complete the
+        information below. BC Registries staff will review your application and if approved, will provide access.
+      </p>
+      <CautionBox class="mt-8" :setMsg="`This is intended for the exclusive use of B.C. ${getMhrSubProduct} only.`" />
+    </section>
+
+    <section class="qs-service-agreement mt-8">
+      <h2>Service Agreement</h2>
+      <p class="mt-1">
+        Read and accept the terms of the service agreement. You may download and a copy of this agreement to save for
+        your records.
+      </p>
+      <v-btn outlined color="primary" class="mt-2" :ripple="false">
+        <img alt="" src="@/assets/svgs/pdf-icon-blue.svg" />
+        <span class="pl-1">Download Qualified Suppliers Agreement</span>
+      </v-btn>
+    </section>
+
+    <section class="qs-information-form mt-8">
+      <h2>Qualified Supplier ({{ getMhrSubProduct }}) Information</h2>
+      <p class="mt-1">
+        Provide the name and contact information for the Qualified Supplier.
+      </p>
+
+      <FormCard label="Qualified Supplier" :showErrors="showErrors" :class="{'border-error-left': showErrors}">
+        <template v-slot:formSlot>
+          <PartyForm
+            ref="qsInformationRef"
+            :baseParty="getMhrQsInformation"
+            :schema="PartyFormSchema"
```

Comment:
@dimak1 You mentioned your preference for schemas earlier, so i did up an example. 
In this implementation, we build out 'custom' schemas using common compound input field rules. 

See the PartyForm for it's implementation and PartyFormSchema for an example of how it's put together.

Reply:
Yes, that's what I was taking about, very nice, thank you! 🎸  ⭐ 
The only small thing I would change is the prop name to be something like validationSchema or rules..